src/Components/Assets/AssetsList.tsx

```diff
@@ -211,6 +212,21 @@ const AssetsList = () => {
     }
   };
 
+  const checkValidAssetId = async (assetId: any) => {
+    const assetData: any = await dispatch(getAsset(assetId));
+    try {
+      if (assetData.data) {
+        navigate(`/assets/${assetId}`);
+      }
+    } catch (err) {
+      console.log(err);
```

Comment:
do remove the console log if it is unintentional

Reply:
@khavinshankar the "err" variable is necessary as per syntax and inside catch block if I don't use "err" then I will be getting eslint errors and warnings.